packages/nodes/table/src/components/TableCellElement/TableCellElementResizable.tsx

```diff
@@ -18,6 +19,19 @@ export type TableCellElementResizableProps = HTMLPropsAs<'div'> &
     step?: number;
   };
 
+const VerticalHandle = ({ colIndex }: { colIndex: number }) => {
+  const setHoveredColIndex = useTableStore().set.hoveredColIndex();
+
+  return (
+    // eslint-disable-next-line jsx-a11y/mouse-events-have-key-events
+    <div
+      style={{ width: '100%', height: '100%' }}
+      onMouseOver={() => setHoveredColIndex(colIndex)}
+      onMouseOut={() => setHoveredColIndex(null)}
+    />
+  );
+};
```

Comment:
We can export this as a headless component (using `createComponentAs` + props hook)